src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/InputModifier.java

```diff
@@ -0,0 +1,133 @@
+////////////////////////////////////////////////////////////////////////////////
+// Test case file for checkstyle.
+// Created: 2001
+////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle.checks.sizes.methodlength;
+
+strictfp final class InputModifier // illegal order of modifiers for class
```

Comment:
input should be named after it's test.
Test is `MethodLengthCheckTest`, input should be named `InputMethodLength...`.
Please rename class and file.